wpiunits/src/generate/main/java/Measure-interface.java.jinja

```diff
@@ -43,12 +43,6 @@ public interface {{ helpers['type_decl'](name) }} extends Measure<{{ helpers['mt
     return ({{ helpers['type_usage'](name) }}) unit().ofBaseUnits(0 - baseUnitMagnitude());
   }
 
-  @Override
-  @Deprecated(since = "2025", forRemoval = true)
-  default {{ helpers['type_usage'](name) }} negate() {
-    return ({{ helpers['type_usage'](name) }}) unaryMinus();
-  }
```

Comment:
This is still needed to sharpen the return type. Otherwise `Volts.of(2).negate()` would suddenly return a generic `Measure<VoltageUnit>` instead of the expected `Voltage` type. Having a default implementation in the `Measure` interface is to allow for generic implementations to have a sane default without needing repetition. This is the same approach taken by most of the math operations, which give a sane default implementation for every case but allow for subtypes to override and return a more relevant type when necessary